canonical-web-and-design/multipass.run
# Update run script

## Done
Updated the run script to the latest version. Added a serve endpoint.

## QA
- Run `./run clean` and check there are no errors
- Run `./run build` and check there are no errors
- Run `./run`
- Check the site looks ok
- Run `./run test` and see it passes
